sdks/wasm/debug.html

```diff
@@ -20,7 +20,7 @@
 			},
 
 			init: function () {
-				this.int_add = Module.mono_bind_static_method ("[sample] Math:IntAdd");
+				this.int_add = Module.mono_bind_static_method ("[sample] DebuggerTests.ValueTypesTest:MethodWithLocalStructs");
```

Comment:
this breaks the functionality of the actual page

Reply:
I can drop this change.